example/rollup.background.js

```diff
@@ -2,7 +2,7 @@ import config from '../rollup.config.js';
 
 
 export default {
-  ...config,
+  ...config[2],
```

Comment:
this syntax looks pretty odd. could you please put it on a variable with a name that describes what it is?